flir.md

```diff
@@ -0,0 +1,43 @@
+# Infrared heat imaging data
+
+Infrared heat imaging data is collected using a FLIR SC615 thermal sensor.
+
+### Level
+
+Raw FLIR thermal imaging output is Level 0 data.
+
+### Sensor information
+
+* Gantry information \(link to Lemnatec documentation for sensor\)
+* [Vendor datasheet](https://terraref.ncsa.illinois.edu/clowder-dev/files/57eae632e4b00b25cabfa4a5?dataset=57eae625e4b00b25cabfa4a1&space=)
+* [Calibration certificate](https://terraref.ncsa.illinois.edu/clowder-dev/files/57eae632e4b00b25cabfa4ad?dataset=57eae625e4b00b25cabfa4a1&space=)
+
+### Data format
+
+* Description of raw data format
+
+* Description of sensor variable metadata
```

Comment:
what about also sensor fixed metadata? How hard would it be to call and parse the sensor metadata from the json files we are using to store it?